### PR TITLE
N41: Add Inox2d

### PR DESCRIPTION
**Pull request by fianathedevgirl**
Note: While Inochi2d has been mainly used for vtubing, its well-established proprietary competitor Live2d is not only used for vtubing but also widely used within the the Japanese game development industry. Games like Hatsune Miku: Colorful Stage, BangDream, Azur Lane, and many others use Live2d. See https://www.reddit.com/r/gachagaming/comments/d5uby1/what_games_do_you_know_that_use_live2d/

_Part of #1233_

### CODE REVIEW

File: content/news/041/index.md [hunk 1]
```diff
@@ -77,6 +77,29 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Library Updates
 
+### [Inox2d](https://github.com/Inochi2D/inox2d/)
+
+![The Japanese character for "inochi" set into a sprocket](https://raw.githubusercontent.com/Inochi2D/inox2d/main/inox2d_logo.svg)
```
Comment (Speykious) [inline content/news/041/index.md:82]: How about using the repo's social preview? I think it'll render better.

```suggestion
![The Japanese character for "inochi" set into a sprocket](https://repository-images.githubusercontent.com/566317952/7b02910a-a581-407e-80a9-fcedd26bc8b9)
```

**Review by Speykious — CHANGES_REQUESTED** [review]
:D

File: content/news/041/index.md (hunk 1 above)
Comment (fianathedevgirl) [reply content/news/041/index.md:82]: Kindaa, SVG is rendered client-side.

Comment (Speykious) [reply content/news/041/index.md:82]: I mean yeah that's true, I was more thinking of the background.

**Review by ozkriff — APPROVED** [review]
:+1: (I've made a couple of fmt/style tweaks for consistency with the regular newsletter style and also embedded the image into the repo for future-proofing

**Comment by ozkriff** [comment]
Tnx for the PR!

File: content/news/041/index.md (hunk 1 above)
Comment (ozkriff) [reply content/news/041/index.md]: @fianathedevgirl Oh, I missed the discussion about the image. Do you want it to be changed? The original svg looks quite fine to me

**Pull request merged**